### PR TITLE
Update Set-CsOnlineVoicemailUserSettings.md

### PR DESCRIPTION
**Pull request by CLYVR**
Updated for new languages and ordered languages.

### CODE REVIEW

File: skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md [hunk 1]
```diff
@@ -13,20 +13,29 @@ ms.reviewer:
...
+By default, users enabled for Phone System will be enabled for Online Voicemail. The Online Voicemail policy controls whether or not voicemail transcription, profanity masking for the voicemail transcriptions, translation for the voicemail transcriptions, and editing call answer rule settings are enabled for a user. The policies also specify the voicemail maximum recording length for a user and the primary and secondary voicemail system prompt languages. 
+
+- Online Voicemail transcription is enabled by default
+- Transcription profanity masking is disabled by default
+- Transcription translation is enabled by default
+- Editing call answer rule settings is enabled by default
+- Voicemail maximum recording length is set to 5 minutes by default
+- Primary and secondary system prompt languages are set to null by default and the users voicemail language setting is used
+ 
+Tenant admin would be able to create customized online voicemail policy to match the organization's requirements.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md:38]:
```suggestion
- Online Voicemail transcription is enabled by default
- Transcription profanity masking is disabled by default
- Transcription translation is enabled by default
- Editing call answer rule settings is enabled by default
- Voicemail maximum recording length is set to 5 minutes by default
- Primary and secondary system prompt languages are set to null by default and the user's voicemail language setting is used
 
Tenant admin would be able to create a customized online voicemail policy to match the organization's requirements.
```

File: skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md [hunk 2]
```diff
@@ -136,6 +145,39 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PrimarySystemPromptLanguage
+The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md:149]:
```suggestion
The primary (or first) language that voicemail system prompts will be presented in. Must also set SecondarySystemPromptLanguage. When set, this overrides the user language choice. Please see [Set-CsOnlineVoicemailUserSettings](/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```

File: skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md [hunk 3]
```diff
@@ -136,6 +145,39 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SecondarySystemPromptLanguage
+The secondary language that voicemail system prompts will be presented in. Must also set PrimarySystemPromptLanguage and may not be the same value as PrimarySystemPromptanguage. When set, this overrides the user language choice.  Please see [Set-CsOnlineVoicemailUserSettings](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md:165]:
```suggestion
The secondary language that voicemail system prompts will be presented in. Must also set PrimarySystemPromptLanguage and may not be the same value as PrimarySystemPromptanguage. When set, this overrides the user language choice.  Please see [Set-CsOnlineVoicemailUserSettings](/powershell/module/skype/set-csonlinevoicemailusersettings?view=skype-ps) -PromptLanguage for supported languages.
```

**Review by dariomws — APPROVED** [review]
It seems that these parameters are also available in Set-CsOnlineVoicemailPolicy. Thanks!

File: skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md (hunk 2 above)
Comment (dariomws) [reply skype/skype-ps/skype/New-CsOnlineVoicemailPolicy.md:149]: Links to articles on docs.microsoft.com must use relative links (/folder/topic-file-name) and not absolute links (https://docs.microsoft.com/folder/topic-file-name).

**Review by get-itips — APPROVED** [review]
Hello @CLYVR Colin, are these parameters going to be released in the next version? I don't see them currently in `2.3.1` but I am ok if they are going to be available soon.

**Review by tseward — APPROVED** [review]

**Comment by get-itips** [comment]
> @get-itips they are available. I could create a new policy using them on 2.3.1.

For some reason the autocomplete for those parameters does not work.

**Comment by dariomws** [comment]
Yes, noticed that.

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this Pr. Thanks

**Pull request merged**